components/control/Input/Input.css

```diff
@@ -88,13 +88,11 @@ diamond-input {
 
   /* Prefix */
   &:has(* + :is(input, textarea, select)) > :first-child {
-    margin-right: calc(var(--diamond-input-padding-inline) / 2 * -1);
```

Comment:
Before this change, the icon took up less room.

## Before

<img width="611" alt="Screenshot 2024-08-21 at 15 41 19" src="https://github.com/user-attachments/assets/a4d8bf53-460c-4227-8d5f-f941ad507e21">

<img width="295" alt="Screenshot 2024-08-21 at 15 44 15" src="https://github.com/user-attachments/assets/1f95ddeb-cb2f-4e90-807a-b6f9c7b12ba5">


## After

<img width="576" alt="Screenshot 2024-08-21 at 15 43 06" src="https://github.com/user-attachments/assets/e89efb8e-f93d-4f07-b9df-f7922f8684dc">

<img width="274" alt="Screenshot 2024-08-21 at 15 44 32" src="https://github.com/user-attachments/assets/480854f6-fd71-4785-aa4b-23f277b7ee9e">

Reply:
I think we should probably just let the icon handle all the padding by adding inline padding to the icon then removing padding from the input on the side where the icon is.

```
  /* Suffix */
  :is(input, textarea, select) + * {
    padding: 0 calc(var(--diamond-input-padding-inline) / 2);
  }

  :is(input, textarea, select):has(+ *) {
    padding-right: 0;
  }
```

I think some special handling is needed for selects, at the moment clicking the select dropdown icon doesn't open the select but it should do.